src/main/java/org/mskcc/oncokb/scheduler/ScheduledTask.java

```diff
@@ -0,0 +1,68 @@
+package org.mskcc.oncokb.scheduler;
+
+import com.mongodb.client.MongoDatabase;
+import org.bson.Document;
+import org.mskcc.oncokb.service.util.MongoUtil;
+import org.mskcc.oncokb.service.util.PythonUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import java.util.List;
+import java.util.TimerTask;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Component
+public class ScheduledTask extends TimerTask {
+
+    private static final Logger log = LoggerFactory.getLogger(ScheduledTask.class);
+
+
+    @Value("${spring.data.mongodb.uri}")
+    private String uri;
+
+    @Value("${application.matchengine.absolute-path}")
+    private String matchengineAbsolutePath;
+
+    @Autowired
+    private MongoDatabase mongoDatabase;
+
+    // Add your task here
+    @Override
+    @Scheduled(fixedRate = 86400000) // rerun MatchEngine every 24 hours
```

Comment:
We should schedule the task at a certain time of the day, not in 24 hours.

Reply:
I'll set it to rerun 24:00 everyday.